tools/add_triage_signature.py

```diff
@@ -839,6 +839,56 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class DownloadImage(ErrorSignature):
+    image_download_regex = re.compile(r"Host (.+?): New image status (.+?). result:.+?; download rate: (.+?) MBps")
+    minimum_download_rate_mb = 10
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="slow_image_download",
+            comment_identifying_string="h1. Host slow download rate:",
+        )
+
+    @classmethod
+    def _list_image_download_info(cls, events):
```

Comment:
I think this information is available in backend metadata.json, no need to parse events

Reply:
(.hosts[].images_status)

Reply:
For some reason this info in the cluster metadata is partial, we show the download rate only for some of the images.
check http://assisted-logs-collector.usersys.redhat.com/files/2023-01-13_13-56-56_158262a6-acea-4b94-9a71-bb2403675007/metadata.json

Reply:
Can you fix that / open a ticket to look into it?

Reply:
It doesn't bother anyone because we have [a dummy threshold](https://github.com/openshift/assisted-service/blob/5cc9607ca5e8154e4683fde764136fbe99a2c149/internal/host/validator.go#L55) for the validation

We get it from the image download [step reply](https://github.com/openshift/assisted-service/blob/76aab2c5d84a1833f3a4eeb10315af9f229ca18d/internal/bminventory/inventory.go#L3339-L3343)

Reply:
Looks like it's [here](https://github.com/openshift/assisted-test-infra/blob/962eed41792c2aff99adc616edbf409872478de8/src/assisted_test_infra/download_logs/download_logs.py#L328), it doesn't seem like we are doing something special for images_status.